src/polymer-cli.ts

```diff
@@ -67,6 +68,17 @@ function parseCLIArgs(commandOptions: any): {[name: string]: string} {
   return parsedOptions;
 }
 
+/**
+ * Shallowly copies an object, converting keys from dash-case to camelCase.
+ */
+function objectDashToCamelCase(input: Object): Object {
+  const output = {};
+  for (const key of Object.keys(input)) {
+    (<any>output)[dashToCamelCase(key)] = (<any>input)[key];
```

Comment:
1) when needing casts, prefer `as`: (output as any)[dashToCamelCase(key)] = (input as any)[key];
2) You shouldn't need the casts if you just type things as `any`

Reply:
Ok, `input` and `output` are now typed as `any`.